source/tutorials.rst

```diff
@@ -103,14 +122,10 @@ The WEC-Sim Simulink model is created by dragging and dropping blocks from the *
    :width: 400pt
    :align: center
 
-.. Note::
-
-	When setting up a WEC-Sim model, it is very important to note the base and follower frames.
-
 
 Step 3: Write wecSimInputFile.m
 """"""""""""""""""""""""""""""""""""""""""""""""
-The WEC-Sim input file defines simulation parameters, body properties, joints, and mooring for the RM3 model. The ``wecSimInputFile.m`` for the RM3 is provided in the RM3 case directory, and shown below.
+The WEC-Sim input file defines simulation parameters, body properties, joints, and mooring for the RM3 model. The ``wecSimInputFile.m`` for the RM3 is provided in the RM3 case directory, and shown below. It is recommended that new users copy information from RM3_wecSimInputFile.m to wecSimInputFile.m, so that they get familiar with the set-up parameters and the files being called in a basic WEC-Sim run.
 
```

Comment:
This relates to my previous comment about the examples versus the tutorial. What do you think?